repo/packages/K/kafka-portworx/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Apache Kafka",
+  "framework": true,
+  "maintainer": "support@mesosphere.io",
+  "minDcosReleaseVersion": "1.8",
```

Comment:
`minDcosReleaseVersion` should be min version of dc/os that px will support

Reply:
This should also be in sync with the Mesosphere SDK-based versions.  (All SDK packages will GA with 1.9.)